<launch>
    <!-- 启动simulator节点 -->
    <!-- pkg：运行哪个pkg，type：可执行文件的名称，而name则是可以任意给出的，它覆盖了原有文件中ros::init指定的node name -->
    <!-- <arg name="FollowMethod" default="PurePursuit" /> -->
    <!-- <arg name="FollowMethod" default="Stanley" />-->
    <arg name="FollowMethod" default="LQR" /> 


    <!-- <arg name="VehicleConfig" default="miniEV" /> -->
    <arg name="VehicleConfig" default="Tesla" />

    <node name="pathfollower" pkg="pathfollower" type="pathfollower_node" output="screen" >
        <!-- drivemode 0:manul 1:auto 2:ipgdrive -->
        <param name="follow_method"  type="string" value="$(arg FollowMethod)" />
        <param name="vehicle_physical_config"  type="string" value="$(find pathfollower)/config/vehicle_physical_$(arg VehicleConfig).yaml"/>
        <param name="lqr_config"  type="string" value="$(find pathfollower)/config/lqr_parameters.yaml"/>

        <!-- <param name="des_ip" type="string" value="192.168.100.1" />
        <param name="sou_ip" type="string" value="192.168.100.100" /> -->
    </node>
</launch>
